insights/specs/default.py

```diff
@@ -140,7 +140,7 @@ class DefaultSpecs(Specs):
     avc_cache_threshold = simple_file("/sys/fs/selinux/avc/cache_threshold")
     aws_instance_id_doc = simple_command("/usr/bin/curl -s http://169.254.169.254/latest/dynamic/instance-identity/document --connect-timeout 5", deps=[IsAWS])
     aws_instance_id_pkcs7 = simple_command("/usr/bin/curl -s http://169.254.169.254/latest/dynamic/instance-identity/pkcs7 --connect-timeout 5", deps=[IsAWS])
-    awx_manage_check_license = simple_command("/usr/bin/awx-manage check_license")
+    awx_manage_check_license = awx_manage.awx_manage_check_license_data
```

Comment:
As the comment above, better add `filterable=True` to its definiation in the `insights/specs/__init__.py`